### DISCUSSION TITLE
Web3Modal Android Kotlin

### DISCUSSION THREAD

**Discussion by muhammad-farhan-bakht** [WalletConnect/walletconnect-monorepo#3466] [Developer Support] [answered]
Hi there,

I am currently integrating Wallet Connect Web3Modal into our Android product. However, I noticed that the official website for Wallet Connect has been updated and there is no longer an option for Web3Modal for Android and iOS.

Could someone please point me to the official documentation for Native Android and iOS? Or has Wallet Connect stopped working on Native Android and iOS? Or the only option is to go Github repo [WalletConnectKotlinV2](https://github.com/WalletConnect/WalletConnectKotlinV2) & [WalletConnectSwiftV2](https://github.com/WalletConnect/WalletConnectSwiftV2) and learn from there?

The following link was working one week ago:

https://docs.walletconnect.com/2.0/walletconnectmodal/usage

Thanks in advance for your help!

Best regards,
Muhammad Farhan

**Comment by radeknovis** [comment, marked as answer]
@muhammad-farhan-bakht  Our docs were restructured recently to accommodate more things. WalletConnectModal docs for all platforms can now be found here https://docs.walletconnect.com/2.0/advanced/walletconnectmodal/about, and in there, you can choose docs for specific platform/language.

**Reply by muhammad-farhan-bakht** [reply]
@radeknovis Thanks for your prompt answer. I see it's the same one. Can you please confirm for me if is there any difference between Web3Modal and WalletConnectModal?
